docs/docs/building-and-running.md

```diff
@@ -110,6 +110,20 @@ kubectl apply -f deploy/kubernetes/local-storage-volume.yaml
 
 Note: If using docker desktop for Windows older than version 19, change the value from "docker-desktop" to "docker-for-desktop" in nodeAffinity in file deploy/kubernetes/local-storage-volume.yaml
 
+### Install the CKAN connector
+
+The Magda CKAN connector (as of v0.0.57) lives outside of the core repository, at https://github.com/magda-io/magda-ckan-connector.
```

Comment:
Should probably say why this is important :)

Reply:
Well, why is it important? :see_no_evil:

Reply:
😂 because whoever runs this is going to need it when they do their first deploy.

I'm thinking something like "You'll need this in order to populate your magda instance with an initial set of datasets from data.gov.au. If you don't want to get datasets from data.gov.au, you can skip this and set `connectors.initialCrawl` (or whatever it is? I'm not sure) to `false` in `config.yaml`."

Reply:
Added a note on this